### PR TITLE
Improve postproc for dry held-suarez

### LINKED ISSUE #421: Make better plots in post processing for held-suarez
To compare with this [paper](https://journals.ametsoc.org/view/journals/bams/75/10/1520-0477_1994_075_1825_apftio_2_0_co_2.xml).

### PR DESCRIPTION
**Pull request by jiahe23**
This postproc is only turned on when the simulation time is greater or equal to 400 days. The first 200 days are thrown away for the statistics computation.

It produces plots comparable to Figs 1-3 in [this paper](https://journals.ametsoc.org/view/journals/bams/75/10/1520-0477_1994_075_1825_apftio_2_0_co_2.xml?tab_body=pdf). Fig 4 needs additional capability of spectra calculation, which will be added later.

closes #421

### CODE REVIEW

File: examples/hybrid/driver.jl [hunk 1]
```diff
@@ -375,6 +375,8 @@ if !is_distributed
         paperplots_baro_wave(sol, output_dir, p, FT(90), FT(180))
     elseif is_column_radiative_equilibrium(parsed_args)
         custom_postprocessing(sol, output_dir)
+    elseif held_suarez == "held_suarez" && t_end >= (3600*24*400)
```
Comment (charleskawczynski) [inline examples/hybrid/driver.jl:378]:
```suggestion
    elseif forcing == "held_suarez" && t_end >= (3600*24*400)
```

File: examples/hybrid/define_post_processing.jl [hunk 2]
```diff
@@ -135,9 +136,22 @@ paperplots_baro_wave(::Val{:ρe}, ::Val{:dry}, args...) =
...
+    paperplots_dry_held_suarez_ρe(args...)
+paperplots_held_suarez(::Val{:ρe_int}, ::Val{:dry}, args...) =
+    paperplots_dry_held_suarez_ρe_int(args...)
+paperplots_held_suarez(::Val{:ρe}, ::Val{:equil}, args...) =
+    paperplots_moist_held_suarez_ρe(args...)
+
 # plots in the Ullrish et al 2014 paper: surface pressure, 850 temperature and vorticity at day 8 and day 10
 function paperplots_baro_wave_ρθ(sol, output_dir, p, nlat, nlon)
-    (; ghost_buffer, ᶜts, ᶜp, params) = p
+    (; ᶜts, ᶜp, params) = p
```
[the remark this answers is not in the document]
Comment (jiahe23) [reply examples/hybrid/define_post_processing.jl:154]: Yes, there used to be a `comms_ctx` unpacked here and later used in dss; but one of the previous PR replaced `comms_ctx` by `ghost_buffer` and removed it from dss, which means it is not used anywhere in this function.

File: examples/hybrid/define_post_processing.jl [hunk 3]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            T_timeave_zonalave',
+            levels = 190:10:310,
+            clim = (190, 310),
+            contour_labels = true,
+            yaxis = :log,
+            title = "u",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:810]:
```suggestion
            title = "T",
```

File: examples/hybrid/define_post_processing.jl [hunk 4]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            θ_timeave_zonalave',
+            levels = 260:10:360,
+            clim = (260, 360),
+            contour_labels = true,
+            yaxis = :log,
+            title = "u",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:826]:
```suggestion
            title = "theta",
```

File: examples/hybrid/define_post_processing.jl [hunk 5]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+            xlabel = "lat (deg N)",
+            ylabel = "z (m)",
+        ),
+        output_dir * "/hs-θ.png",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:830]:
```suggestion
        output_dir * "/hs-theta.png",
```
(I just prefer no unicode in filenames, but that's up to you)

File: examples/hybrid/define_post_processing.jl [hunk 6]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            T_timeave_zonalave',
+            levels = 190:10:310,
+            clim = (190, 310),
+            contour_labels = true,
+            yaxis = :log,
+            title = "u",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:979]:
```suggestion
            title = "T",
```

File: examples/hybrid/define_post_processing.jl [hunk 7]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            θ_timeave_zonalave',
+            levels = 260:10:360,
+            clim = (260, 360),
+            contour_labels = true,
+            yaxis = :log,
+            title = "u",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:995]:
```suggestion
            title = "theta",
```

File: examples/hybrid/define_post_processing.jl [hunk 8]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+            xlabel = "lat (deg N)",
+            ylabel = "z (m)",
+        ),
+        output_dir * "/hs-θ.png",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:999]:
```suggestion
        output_dir * "/hs-theta.png",
```

File: examples/hybrid/define_post_processing.jl [hunk 9]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            T_timeave_zonalave',
+            levels = 190:10:310,
+            clim = (190, 310),
+            contour_labels = true,
+            yaxis = :log,
+            title = "u",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:1147]:
```suggestion
            title = "T",
```

File: examples/hybrid/define_post_processing.jl [hunk 10]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            θ_timeave_zonalave',
+            levels = 260:10:360,
+            clim = (260, 360),
+            contour_labels = true,
+            yaxis = :log,
+            title = "u",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:1163]:
```suggestion
            title = "theta",
```

File: examples/hybrid/define_post_processing.jl [hunk 11]
```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
...
+            xlabel = "lat (deg N)",
+            ylabel = "z (m)",
+        ),
+        output_dir * "/hs-θ.png",
```
Comment (szy21) [inline examples/hybrid/define_post_processing.jl:1167]:
```suggestion
        output_dir * "/hs-theta.png",
```

**Review by szy21 — APPROVED** [review]

**Comment by jiahe23** [comment]
good catch! @szy21

**Pull request merged**